server/sonar-db-dao/src/main/java/org/sonar/db/component/ComponentDao.java

```diff
@@ -321,6 +326,10 @@ public void update(DbSession session, ComponentUpdateDto component) {
     mapper(session).update(component);
   }
 
+  public void updateTags(DbSession session, ComponentDto component) {
```

Comment:
You can forget, I didn't understand the goal of these columns, now I understand that you have nothing to that about that.